### DISCUSSION TITLE
Support for remote development

### DISCUSSION THREAD

**Discussion by darrenwsun** [kdbinsidebrains/plugin#72] [Ideas]
I use remote development under IDEA Ultimate, and when trying to add new scope to add KDB connection, the scope wasn't added after filling required info. The issue is not reproducible in local development mode.

It may be just because the plugin doesn't fully support Intellij Gateway/Client (what's being used during remote development). If so, is it a major effort to support them? It might also be due to other reasons I'm not aware of...

**Comment by smklimenko** [comment]
hm, don't think I can check that in near future. I don't need that and have no environment to test as Linux server is required for the server part. Could be the plugin doesn't work under Linux, as I've never tested it there.
